becca-new-app/src/App.js

```diff
@@ -2,29 +2,57 @@ import React, { useEffect, useState } from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
-const useSemiPersistentState = () => {
-	const initialTodoList =
-		JSON.parse(localStorage.getItem("savedTodoList")) || [];
-	const [todoList, setTodoList] = useState(initialTodoList);
+function App() {
+	const [todoList, setTodoList] = useState([]);
+	const [isLoading, setIsLoading] = useState(true);
 
 	useEffect(() => {
-		localStorage.setItem("savedTodoList", JSON.stringify(todoList));
-	}, [todoList]);
+		const fetchData = async () => {
+			try {
+				const dataPromise = new Promise((resolve, reject) => {
+					setTimeout(() => {
+						resolve({ data: { todoList: [] } });
```

Comment:
I'm not getting any data after it is done loading. Look into this line, what do you need to change?